Content.Server/StationEvents/Events/AnomalySpawnRule.cs

```diff
@@ -16,7 +16,7 @@ protected override void Added(EntityUid uid, AnomalySpawnRuleComponent component
 
         var str = Loc.GetString("anomaly-spawn-event-announcement",
             ("sighting", Loc.GetString($"anomaly-spawn-sighting-{RobustRandom.Next(1, 6)}")));
-        ChatSystem.DispatchGlobalAnnouncement(str, colorOverride: Color.FromHex("#18abf5"));
+        ChatSystem.DispatchGlobalAnnouncement(str, playSound: false, colorOverride: Color.FromHex("#18abf5"));
```

Comment:
You're right, I thought about it, but forgor

Reply:
`StationEventSystem` `Add` method calls before GameRules systems, so their component values cannot be overwritten and used properly except if we make another event.

Reply:
> uh im not sure i understand your issue

While debugging I've found out that the overrided "Add" method in `StationEventSystem` was called before I could change its component values for example in `AnomalySpawnRuleSystem`. They both use the same method that being called when "GameRuleAddedEvent" rised. I guess that it's because the `StationEventComponent` is being serialized before others.

Reply:
aight, push your changes and i'll see if i can do something these days to fix that problem cuz that shouldnt really happen

Reply:
> aight, push your changes and i'll see if i can do something these days to fix that problem cuz that shouldnt really happen

Already pushed in [a5e532b](https://github.com/space-wizards/space-station-14/pull/28331/commits/a5e532bd5858c364efb57ae88d2cf84a5b020d8a)

Reply:
Well you just had to call base.Added at the end of the method instead of the start (base calls the class that you are overriding so you were making the announcement with the default message and THEN you overrode the message)